vmm-reference-queue-handler.md

```diff
@@ -0,0 +1,92 @@
+# [Queue Handler](./../src/devices/src/virtio/net/queue_handler.rs)
```

Comment:
Change the link to absolute. It doesn't work rn

Reply:
Done